packages/design-system/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "@stanfordbdhg/design-system",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "build": "tsc && tailwindcss -i ./src/main.css -o ./dist/main.css",
+    "watch": "concurrently \"tsc -w\" \"tailwindcss -i ./src/main.css -o ./dist/main.css --watch\"",
+    "storybook": "storybook dev -p 6006",
```

Comment:
Run storybook locally! In the future we could have GH pages with build.

![image](https://github.com/StanfordBDHG/ENGAGE-HF-Web-Frontend/assets/60391032/38822fed-5c0a-4475-aa1f-097f013fb908)

Reply:
https://github.com/StanfordBDHG/ENGAGE-HF-Web-Frontend/issues/3